cpp/src/parquet/column_reader.h

```diff
@@ -218,9 +218,11 @@ class TypedColumnReader : public ColumnReader {
                                   int64_t valid_bits_offset, int64_t* levels_read,
                                   int64_t* values_read, int64_t* null_count) = 0;
 
-  // Skip reading levels
-  // Returns the number of levels skipped
-  virtual int64_t Skip(int64_t num_rows_to_skip) = 0;
+  // Skip reading values.
+  // Returns the number of values skipped.
+  // This function will NOT skip rows, and repeated fields may have multiple values
+  // corresponding to the same row.
```

Comment:
I find this comment confusing. Let's say I ask to skip 20 values:
* If I am in a non-repeated field, nothing is skipped and 0 is returned?
* If I am in a repeated field and the current row has 10 values, 10 is returned? How do I skip several rows?

Reply:
I clarified the comment, please take a look and see if it makes it clear. Thanks!